code/modules/urist/mob/uristanimals.dm

```diff
@@ -69,7 +69,7 @@ Please keep it tidy, by which I mean put comments describing the item before the
 
 //ARANEUS. Squee?
 
-/mob/living/simple_animal/hostile/retaliate/bat/araneus
+/mob/living/simple_animal/hostile/retaliate/spider/araneus
```

Comment:
while araneus does not need to be a bat to avoid spawnkilling the HoS, you may want to make sure it still has the bat-traits it was inheriting like attacktext, response_harm/disarm, faction, maybe see_in_dark. all spiders are currently just hostile/giant_spider also, so araneus has nothing to inherit from /spider/ here and remains a pretend spider

Reply:
yeah, Araneus was always a weird bat subtype because of being a retaliate creature, this is intentional and doesn't need to be fixed unless there's some issues being caused by inheriting weirdness from being a bat that have been missed in the last 10 years or so since Araneus was added (very possible)

Reply:
I thought it was accidental change from baymerge, makes sense though I'll change it back